src/definitions_cxx.hpp

```diff
@@ -620,6 +620,13 @@ enum class ArpOctaveMode {
 	RANDOM,
 };
 
+enum class ArpMpeModSource {
+	OFF,
+	AFTERTOUCH,
+	MPE_Y,
+	MPE_X,
```

Comment:
I'm not sure about this - I think the current behaviour of bending the note is better. If there's an edge case where it doesn't work we should fix it

Reply:
I can remove it. Actually it would be weird to have bending and at the same time some modulation. We can leave it just as aftertouch or Y for all.params

Reply:
Yeah I agree

Reply:
changes done